examples/simpleSwitch.js

```diff
@@ -0,0 +1,62 @@
+const readline = require('readline')
+const SerialPort = require('serialport')
+const EnoceanParser = require('../ESP3Parser').FullParser
+const Packet = require('../Packet.js')
+const port = new SerialPort('/dev/ttyUSB0', { baudRate: 57600 })
+const parser = port.pipe(new EnoceanParser())
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+var rocker
+
+init()
+rl.on('line', async (input) => {
+  var mode = 0x10
+  if (input === 'off') mode = 0x30
+  // send button press event
+  rocker.payload = mode
+  res = await sendAndWaitForResponse(rocker.getRawBuffer())
+  // send button release event
```

Comment:
most commonly, you implement some kind of sensor that does not send its data just once. see this light switch  for example. Here a packet is prepared, and everything in this packet stays the same, only 1 Byte ever changes. 
Even for a single click you have to send  two packets: button press and release.
the purpose of the getters and setter is to keep the packet data in sync whenever something changes. 

What would these lines ideally look like with you API?